.github/workflows/conformance.yml

```diff
@@ -0,0 +1,34 @@
+name: Ruby Conformance CI
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        ruby-version: [2.5, 2.6, 2.7]
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.15'
+    - name: Setup Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+    - name: Bundle install
+      run: 'bundle install'
+    - name: Run HTTP conformance tests
+      uses: GoogleCloudPlatform/functions-framework-conformance/action@v0.3.6
```

Comment:
Is it expected that we need to call out the specific version? (i.e. is this something we'll have to keep updated?)

Reply:
As far as I can tell you have to specify some version, so I'll change it to v0.3.

Reply:
Turns out that doesn't work. It'll have to be pinned for now.